lib/cisco_node_utils/acl.rb

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # RouterAcl - node utility class for RouterAcl config mgmt.
+  class Acl < NodeUtil
+    attr_reader :acl_name, :afi
+
+    # name: name of the router instance
+    # instantiate: true = create router instance
+    def initialize(acl_name, afi, instantiate=true)
+      @acl_name = acl_name
+      @afi = afi
+      @set_args = @get_args = { acl_name: @acl_name }
+      create if instantiate
+    end
+
+    def self.acls
+      instances = config_get('acl_v4', 'acl') if @afi == 'v4'
+      instances = config_get('acl_v6', 'acl') if @afi == 'v6'
```

Comment:
1. For consistency with all of our other providers it would be better if you use `ipv4` and `ipv6` for the afi strings.

2. There's a lot of redundancy here. You could simplify these if you set a class variable in intialize and then just use that for each of the getter/setters, ie.:

```
def initialize( ...
  @afi_str = "acl_v#{$1}" if afi[/ipv(\d)/]
   ...
end

def self.acls
  instances = config_get('@afi_str, 'acl')
  ...
```